src/formio/components/licenseplate.ts

```diff
@@ -0,0 +1,35 @@
+import {InputComponentSchema, MultipleCapable} from '..';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export type LicensePlateInputSchema = InputComponentSchema<string, Validator, TranslatableKeys>;
+
+/**
+ * @group Form.io components
+ * @category Base types
+ */
+export interface LicensePlateProperties {
+  validate: {
+    pattern: '^[a-zA-Z0-9]{1,3}\\-[a-zA-Z0-9]{1,3}\\-[a-zA-Z0-9]{1,3}$';
+  };
+  validateOn?: 'blur';
+}
+
+/**
+ * @group Form.io components
+ * @category Base types
+ */
+export type BaseLicensePlateComponentSchema = Omit<
+  LicensePlateInputSchema,
+  'hideLabel' | 'placeholder'
+> &
+  LicensePlateProperties & {
+    type: 'licenseplate';
```

Comment:
is there a reason this is not part of `LicensePlateProperties`?

Reply:
That was taken from postcode, but it also have other fields so yes no reason to not be part of `LicensePlateProperties`